tools/serverpod_cli/test/analyzer/models/stateful_analyzer/model_validation/field/default/string/field_string_default_model_test.dart

```diff
@@ -197,7 +196,7 @@ void main() {
         var definition = definitions.first as ClassDefinition;
         expect(
           definition.fields.last.defaultModelValue,
-          '"This \'is\' a default model value"',
+          '\'This \\\'is\\\' a default model value\'',
```

Comment:
Question: Is this format of the string correctly parsed in Postgres?

Reply:
If the value is defined in the database, would we get `This 'is' a default model value` as string for the object?

Reply:
Done!